bundles/automation/org.eclipse.smarthome.automation.module.core/ESH-INF/automation/moduletypes/EventTriggersTypeDefinition.json

```diff
@@ -110,6 +110,41 @@
                "reference":"itemStateChangeTriggerID.event"
             }
          ]
+      },
+      {
+         "uid":"ThingEventTrigger",
+         "label":"Thing Trigger Event Trigger",
+         "description":"This triggers a rule if a channel emits a trigger",
+         "configDescriptions":[
+            {
+               "name":"channelId",
+               "type":"TEXT",
+               "context":"channel",
+               "label":"channel ID",
```

Comment:
As a label I would simply choose "Channel" (assuming that the UI will anyhow respect the context and render a drop down list with the available channels).